common/src/main/java/dev/isxander/yacl3/api/controller/DropdownStringControllerBuilder.java

```diff
@@ -0,0 +1,17 @@
+package dev.isxander.yacl3.api.controller;
+
+import dev.isxander.yacl3.api.Option;
+import dev.isxander.yacl3.gui.controllers.dropdown.DropdownMode;
+import dev.isxander.yacl3.impl.controller.DropdownStringControllerBuilderImpl;
+
+import java.util.List;
+
```

Comment:
javadoc please!

Reply:
I left out the Javadoc because none of the classes in that package have it